manta-pay/src/key.rs

```diff
@@ -288,6 +291,13 @@ impl PartialEq for Mnemonic {
     }
 }
 
+impl Hash for Mnemonic {
+    #[inline]
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.as_ref().hash(state)
+    }
+}
```

Comment:
Why doesn't `derive(Hash)` work?

Reply:
Because the `PartialEq` implementation is custom